### PR TITLE
[KHWEBA-81]Table-transaction 3dot menu

### PR DESCRIPTION
**Pull request by OlgaOrlova**
image

### CODE REVIEW

File: src/app/transactions/table-transactions/table-transactions.component.ts [hunk 1]
```diff
@@ -23,6 +23,14 @@ export class TableTransactionsComponent {
   ngAfterViewInit() {
     this.dataSource.sort = this.sort;
   }
+
+  editTransaction() {
+    console.log("editTransaction");
```
Comment (gift-a) [inline src/app/transactions/table-transactions/table-transactions.component.ts]: should logic be in transactions? not in table?

**Review by gift-a — DISMISSED** [review]

**Review by Vladyslav-Plakhuta — DISMISSED** [review]

File: src/app/transactions/table-transactions/table-transactions.component.html [hunk 2]
```diff
@@ -2,30 +2,39 @@
...
     </ng-container>
     <ng-container matColumnDef="account">
       <mat-header-cell class="table-transactions__header-cell" *matHeaderCellDef mat-sort-header>
-        Account </mat-header-cell>
-      <mat-cell *matCellDef="let element"> {{element.account}} </mat-cell>
+        Account
+      </mat-header-cell>
+      <mat-cell *matCellDef="let element" class="table-transactions__cell"> {{element.account}}
+        <app-button-more [position]="'before'" class="table-transactions__button-more" (editClick)="editTransaction()"
+          (deleteClick)="deleteTransaction()"></app-button-more>
```
Comment (gift-a) [inline src/app/transactions/table-transactions/table-transactions.component.html]: you beautified previous html. could you beautify this row too?

File: src/app/transactions/table-transactions/table-transactions.component.html [hunk 3]
```diff
@@ -2,30 +2,39 @@
...
+      <mat-cell *matCellDef="let element"> {{element.tag}}</mat-cell>
     </ng-container>
     <ng-container matColumnDef="account">
       <mat-header-cell class="table-transactions__header-cell" *matHeaderCellDef mat-sort-header>
-        Account </mat-header-cell>
-      <mat-cell *matCellDef="let element"> {{element.account}} </mat-cell>
+        Account
+      </mat-header-cell>
+      <mat-cell *matCellDef="let element" class="table-transactions__cell"> {{element.account}}
+        <app-button-more [position]="'before'" class="table-transactions__button-more" [data]="{type: 'transaction', name: element.account}"
```
Comment (gift-a) [inline src/app/transactions/table-transactions/table-transactions.component.html]: You need to send description, not account

File: src/app/transactions/table-transactions/table-transactions.component.html [hunk 4]
```diff
@@ -2,30 +2,39 @@
...
+      <mat-cell *matCellDef="let element"> {{element.tag}}</mat-cell>
     </ng-container>
     <ng-container matColumnDef="account">
       <mat-header-cell class="table-transactions__header-cell" *matHeaderCellDef mat-sort-header>
-        Account </mat-header-cell>
-      <mat-cell *matCellDef="let element"> {{element.account}} </mat-cell>
+        Account
+      </mat-header-cell>
+      <mat-cell *matCellDef="let element" class="table-transactions__cell"> {{element.account}}
+        <app-button-more [position]="'before'" class="table-transactions__button-more" [data]="{type: 'transaction', name: element.description}"
```
Comment (ihor7016) [inline src/app/transactions/table-transactions/table-transactions.component.html:32]: do you have working precommit hooks? this should have been formatted (line is too long)

**Review by ihor7016 — APPROVED** [review]

**Pull request merged**